recipes/coin/meta.yaml

```diff
@@ -0,0 +1,57 @@
+package:
+  name: coin
+  version: 4.0.0
+
+source:
+  fn: coin.tar.gz
+  url: https://bitbucket.org/Coin3D/coin/get/CMake.tar.gz
```

Comment:
What about making a tag with the zipped source once the pr is accepted and the repository is created?